test/helper.js

```diff
@@ -21,14 +21,44 @@ const purgeDlqBySubcriptionId = async subscriptionId => {
 	await bus.processDlq(subscriptionId, accept);
 };
 
+const purgeActiveBySubcriptionId = async (subscriptionId, n) => {
+	let activeBodies;
+	try {
+		activeBodies = await bus.peekActive(subscriptionId, n);
```

Comment:
we name this `deadBodies` because we referred to "dead" letters. In this context I'd choose a different variable name like `activeMessages`

Reply:
Renaming deadBodies to activeMessages now 👍